Contentless/Program.cs

```diff
@@ -43,11 +44,62 @@ public static class Program {
         var excluded = config.ExcludedFiles.Select(Program.MakeFileRegex).ToArray();
         var overrides = Program.GetOverrides(config.Overrides).ToArray();
 
+        var referencesVersions = config.References.ToList().ToDictionary(x => x, x => (string)null, StringComparer.OrdinalIgnoreCase);
+        if (config.References.Any())
+        {
+            var csprojPath = args[1];
```

Comment:
Will crash if we only have one arg, this needs to be made optional for backwards compatibility & users who don't use the targets file for running this

Reply:
Fixed